tests/queries/0_stateless/02710_show_table.sql

```diff
@@ -0,0 +1,15 @@
+DROP TABLE IF EXISTS t_2710_show_table;
+
+CREATE TABLE t_2710_show_table(n1 UInt32, s String) engine=Log;
+SHOW TABLE t_2710_show_table;
+SHOW CREATE TABLE t_2710_show_table;
+SHOW CREATE t_2710_show_table;
+
```

Comment:
The test was flaky when running in CI: https://s3.amazonaws.com/clickhouse-test-reports/48591/e4b1711812b58a12abd648a166825580868e6881/stateless_tests_flaky_check__asan_.html

You either want to add `IF EXISTS` to l. 8 and 11 or mark the test as `no-parallel` (should be okay since it is fast).